bcl2fastq/index_summary.py

```diff
@@ -24,10 +27,16 @@
         id='run_select',
         options=all_runs,
         value=all_runs[0]['value'],
+        clearable=False
     ),
+#           Dropdown option's label set to all_runs(combination of unknown and known)
```

Comment:
By convention, comments are often added above the thing that they refer to, to give context before reading the code. Could this comment be moved above the `layout = ...` declaration?